homeassistant/components/glances/sensor.py

```diff
@@ -300,6 +309,7 @@ def _migrate_old_unique_ids(
                             sensor_description,
                         )
                     )
+    entities.append(GlancesSensor(coordinator, name, "", SENSOR_TYPES["uptime"]))
```

Comment:
This line should not be useful as the previous "for" loop already handles all sensors